### DISCUSSION TITLE
WINDBREAK-SHELTERBELT ESTABLISHMENT AND RENOVATION (380)

### DISCUSSION THREAD

**Discussion by ktoddbrown** [ktoddbrown/CSPStandards#12] [General]
**Definition**: Establishing, enhancing, or renovating windbreaks, also known as shelterbelts, which are single or multiple rows of trees and/or shrubs in linear or curvilinear configurations.

There is both a [national standard](USDA_CSP_pdf/380_Windbreak_Shelterbelt_Establishment_NHCP_CPS_2021.pdf) and a [Pacific Island standard](USDA_CSP_pdf/380_Windbreak_Shelterbelt_Establishment_PIA_CPS_2022.pdf)

Differences as found by the diff of the processed yml are as follows:

- Pacific island standard was drafted in January 2022 and the National standard in July 2021 [diff log 1-4](https://github.com/ktoddbrown/CSPStandards/blob/e8c673493bfd4ee4a08fa51257e9bfcefaace2bd/data_lvl03/380_compare.log#L1)
- National standards address snow management and distributions that are not included in the pacific island standards. [diff log 5-7](https://github.com/ktoddbrown/CSPStandards/blob/e8c673493bfd4ee4a08fa51257e9bfcefaace2bd/data_lvl03/380_compare.log#L5), [diff log 8-11](https://github.com/ktoddbrown/CSPStandards/blob/e8c673493bfd4ee4a08fa51257e9bfcefaace2bd/data_lvl03/380_compare.log#L8), [diff log 12-15](https://github.com/ktoddbrown/CSPStandards/blob/e8c673493bfd4ee4a08fa51257e9bfcefaace2bd/data_lvl03/380_compare.log#L12), [diff log 21-30](https://github.com/ktoddbrown/CSPStandards/blob/e8c673493bfd4ee4a08fa51257e9bfcefaace2bd/data_lvl03/380_compare.log#L21), [diff log 31-34](https://github.com/ktoddbrown/CSPStandards/blob/e8c673493bfd4ee4a08fa51257e9bfcefaace2bd/data_lvl03/380_compare.log#L31), [diff log 35-39](https://github.com/ktoddbrown/CSPStandards/blob/e8c673493bfd4ee4a08fa51257e9bfcefaace2bd/data_lvl03/380_compare.log#L35), [diff log 49-55](https://github.com/ktoddbrown/CSPStandards/blob/e8c673493bfd4ee4a08fa51257e9bfcefaace2bd/data_lvl03/380_compare.log#L49)
- Pacific island standards are required to be identified by vegetative specification for this standard or accepted variance [diff log 16, 20](https://github.com/ktoddbrown/CSPStandards/blob/e8c673493bfd4ee4a08fa51257e9bfcefaace2bd/data_lvl03/380_compare.log#L16), 
- Burning of woody residue is not allowed in Pacific Island standards [diff log 40-43](https://github.com/ktoddbrown/CSPStandards/blob/e8c673493bfd4ee4a08fa51257e9bfcefaace2bd/data_lvl03/380_compare.log#L40)
- Specific instructions for deciduous and coniferous trees in national standards that are not in pacific island standards [diff log 44-48](https://github.com/ktoddbrown/CSPStandards/blob/e8c673493bfd4ee4a08fa51257e9bfcefaace2bd/data_lvl03/380_compare.log#L44)
- Restoration named explicitly in requirements document in Pacific Island standards [diff log 56-59](https://github.com/ktoddbrown/CSPStandards/blob/e8c673493bfd4ee4a08fa51257e9bfcefaace2bd/data_lvl03/380_compare.log#L56)

Overall references to snow are removed from Pacific Island specifications, burning is not allowed in Pacific Islands, species are specifications are difference, and restoration is explicitly named in Pacific islands.

**Comment by 9zr5EHJA** [comment]
Currently working on a variance request to include an additional species in the PIA vegetation spec (approved species list).
